sentry_sdk/utils.py

```diff
@@ -680,6 +680,11 @@ def get_errno(exc_value):
     # type: (BaseException) -> Optional[Any]
     return getattr(exc_value, "errno", None)
 
+def get_error_message(exc_value: BaseException):
+    # type: (BaseException) -> str
+    return (getattr(exc_value, "message", "") 
+            or getattr(exc_value, "detail", "") 
+            )
```

Comment:
I deleted the safe_str function from here. It was conflicting with the BaseException.